tests/cli-app/test/integration.test.js

```diff
@@ -73,6 +73,14 @@ describe(`cli-app on ${network}`, function () {
     truffleExec(`run.js WithToken 0 say --network ${network} --from ${this.from}`).should.eq('TKN')
   })
 
+  it('creates an instance from a dependency from inside a subfolder', function () {
+    run(`cd contracts`)
```

Comment:
Note that `run` automatically sets the workdir, so this call has no effect. The correct way to test it should be adding a new `cwd` option to `run`.

Reply:
Fixed.